CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Unreleased
+- Friends Page: ensure Friends posts can be collapsed and uncollapsed.
```

Comment:
Thanks, this is not needed as I don't do Unreleased changelog entries here but generate them from PRs: https://github.com/akirk/friends/blob/main/bin/release.sh#L41